CriteoPublisherSdk/Sources/Standalone/CRBannerView.m

```diff
@@ -137,12 +130,17 @@ - (void)loadAdWithBidToken:(CRBidToken *)bidToken {
                        @"Token passed to loadAdWithBidToken doesn't have the same ad unit as the CRBannerView was initialized with"];
     return;
   }
-  if (!tokenValue.displayUrl)
-    return [self safelyNotifyAdLoadFail:CRErrorCodeInternalError
-                            description:@"No display URL in bid response"];
+
+  [self displayAdWithData:tokenValue.displayUrl];
+}
+
+- (void)displayAdWithData:(NSString *)displayData {
+  if (!displayData || displayData.length == 0) {
```

Comment:
@TurpIF or @Amokrane can you please confirm that considering an empty `displayUrl` as an error is relevant? (Added that `displayData.length == 0` check)

Reply:
On Android, we just have a `@NonNull` to inform that null is an invalid value (error from Kotlin, warning from Java). So there is no validation of the data. Although, it makes also sense that an empty `displayUrl` is wrong. I don't have a strong opinion on this.

Reply:
Thanks